plato-api/src/main/java/org/techpleiad/plato/api/request/EmailConfigRequestTo.java

```diff
@@ -0,0 +1,20 @@
+package org.techpleiad.plato.api.request;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Getter
+@ToString
+public class EmailConfigRequestTo {
+    private boolean sendEmail;
+    private List<String> recipients;
```

Comment:
Use annotation @ValidEmail

Reply:
Will have to add new library or write custom code for validation of email.